src/ImgCrop.tsx

```diff
@@ -347,34 +350,38 @@ const ImgCrop = forwardRef<CropperRef, ImgCropProps>((props, cropperRef) => {
     <>
       {getNewUpload(children)}
       {modalImage && (
-        <AntModal
-          {...modalProps}
-          {...modalBaseProps}
-          {...{ [openProp]: true }}
-          title={title}
-          onCancel={onCancel.current}
-          onOk={onOk.current}
-          wrapClassName={wrapClassName}
-          maskClosable={false}
-          destroyOnClose
+        <AntConfigProvider 
```

Comment:
We can also check if `theme` prop is present, if don't so there's no need to render `ConfigProvider` and just let it inherit from app's `ConfigProvider`